src/pages/index.js

```diff
@@ -23,18 +29,85 @@ import {
   profileDescriptionInput,
 } from "../utils/constants.js";
 
-/* ----------------------- */
-/*     Profile Edit        */
-/* ----------------------- */
-const profileEditModalFormValidator = new FormValidator(
-  editProfileForm,
-  options
-);
-const profileUserInfo = new UserInfo({
+
+/*-----------------*/
+/*       Api      */
+/*---------------*/
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: authorizationCode,
+  },
+});
+
+
+api.getInitialCards().then((cards) => {
+  cardSection.renderItems(cards);
+  })
+  .catch((err) => {console.error("Error fetching cards:", err);
+});
```

Comment:
You need to format the markup to show **indentation** and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`


```js

api
  .getInitialCards()
  .then((cards) => {
    cardSection.renderItems(cards);
  })
  .catch((err) => {
    console.error("Error fetching cards:", err);
  });
```